changelogs/fragments/add_require_Connection.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - Avoid unnecessary use of a persistent connection.
```

Comment:
Can we please elaborate on the user facing impact of this change and to which plugins this applies?